### PR TITLE
fix: added `WEBGL` mode for correct blendMode(SUBTRACT) behavior

### PR DESCRIPTION
**Pull request by Abhinavcode13**
Resolves - #7217

### CODE REVIEW

File: src/core/rendering.js [hunk 1]
```diff
@@ -1158,23 +1158,25 @@ p5.prototype.clearDepth = function(depth) {
...
+ *   // Set the blend mode to SUBTRACT.
  *   blendMode(SUBTRACT);
  *
  *   // Style the lines.
  *   strokeWeight(30);
  *
  *   // Draw the blue line.
  *   stroke('blue');
- *   line(25, 25, 75, 75);
+ *   line(-25, -25, 25, 25);  // Adjusted coordinates for WEBGL
```
Comment (davepagurek) [inline src/core/rendering.js:1175]: We probably don't need the "adjusted coordinates" comment, these are just the regular coordinates for a WebGL canvas.

**Review by davepagurek — COMMENTED** [review]
Thanks for making these fixes! Just one request on the comments, since users won't have the context of a prior 2D mode version.

File: src/core/rendering.js (hunk 1 above)
Comment (Abhinavcode13) [reply src/core/rendering.js:1175]: Ok done

**Comment by Abhinavcode13** [comment]
gentle ping - @davepagurek

**Review by davepagurek — APPROVED** [review]
Looks good, thanks!

**Comment by davepagurek** [comment]
@all-contributors please add @Abhinavcode13 for docs

**Pull request merged**